crates/transaction-pool/src/ordering.rs

```diff
@@ -18,3 +19,20 @@ pub trait TransactionOrdering: Send + Sync + 'static {
     /// Returns the priority score for the given transaction.
     fn priority(&self, transaction: &Self::Transaction) -> Self::Priority;
 }
+
+/// Default ordering for the pool.
+///
+/// The transactions are ordered by their cost. The higher the cost,
+/// the higher the priority of this transaction is.
+#[derive(Debug, Default)]
+#[non_exhaustive]
+pub struct CostOrdering;
```

Comment:
nit: we should consider making this generic over `T: PoolTransaction` via phantomtype

Reply:
there is a minor issue with that. since `TransactionOrdering` requires that `Self: 'static`, it wants the same lifetime bound imposed on the generic

Reply:
I guess the 'static bound should be removed to make it consistent across the codebase